### PR TITLE
Update past-gen item/move/ability descriptions

### PR DESCRIPTION
**Pull request by DeathlyPlays**

### CODE REVIEW

**Comment by AustinXII** [comment]
You probably could have done these all in one commit

**Comment by Zarel** [comment]
Let me guess, you don't know how to edit multiple files because you're using GitHub's web interface?

**Comment by DeathlyPlays** [comment]
I know how to @Zarel do you want me to make them all in this PR?

**Comment by DeathlyPlays** [comment]
There

**Comment by DeathlyPlays** [comment]
boop @Zarel

**Comment by DeathlyPlays** [comment]
That's all I see e_e

**Comment by Zarel** [comment]
Yes, this is better, thanks! I'm waiting for @Marty-D to review the descriptions.

**Comment by DeathlyPlays** [comment]
Alrighty :3 I think the descriptions are accurate but glad to fix up any incorrect information (sorry I didn't put it in one PR originally I was unsure if I should combine it all since its about all different types of gens)

**Comment by Marty-D** [comment]
Some general thoughts:

Descriptions for past gen stuff are supposed to be the first thing after `inherit: true,`, assuming inheritance is happening. I notice the Gen 3 items don't seem to follow the standard but that's someone else's mistake.

- You left a bunch of references to weather in Gen 1 which aren't needed.
- Ignoring immunity is not an interesting feature of Status moves. In almost every gen, all Status moves except Thunder Wave have that property. It would be more worthwhile to point out when they _don't_ ignore type immunities.
- For Gen 2, you did Stick but not Lucky Punch which is directly above it. Also if you look at the code it returns 3, not +3. They override all other crit-boosting effects.

File: mods/gen3/items.js [hunk 1]
```diff
@@ -215,6 +217,7 @@ exports.BattleItems = {
 		onEat: function (pokemon) {
 			this.heal(30);
 		},
+		desc: "The holder heals 30 HP.",
```
Comment (Marty-D) [inline mods/gen3/items.js]: When?

File: mods/gen3/items.js [hunk 2]
```diff
@@ -120,6 +121,7 @@ exports.BattleItems = {
 				pokemon.removeVolatile('attract');
 			}
 		},
+		desc: "Cures holder of being infatuated. Single use.",
```
Comment (Marty-D) [inline mods/gen3/items.js]: This is already in the Gen 4 file.

File: mods/gen3/items.js (hunk 1 above)
Comment (DeathlyPlays) [inline mods/gen3/items.js]: Is that what this.heal(30) is or am I mistaken

File: mods/gen3/items.js (hunk 2 above)
Comment (DeathlyPlays) [inline mods/gen3/items.js]: Oh ok

**Comment by DeathlyPlays** [comment]
@Marty-D I think that's all the issues you addressed

**Comment by DeathlyPlays** [comment]
Feel free to squash this whenever you merge it haha

File: mods/gen1/moves.js [hunk 3]
```diff
@@ -356,6 +357,8 @@ exports.BattleMovedex = {
 	glare: {
 		inherit: true,
 		ignoreImmunity: true,
+		desc: "Paralyzes the target.",
+		shortDesc: "Paralyzes the target.",
```
Comment (Zarel) [inline mods/gen1/moves.js]: ? Why is this here? This hasn't changed in any of the 7 generations.

File: mods/gen1/moves.js [hunk 4]
```diff
@@ -530,18 +533,22 @@ exports.BattleMovedex = {
 		inherit: true,
 		ignoreImmunity: true,
 		basePower: 1,
+		desc: "Deals damage to the target equal to the user's level. Ignores Immunities.",
+		shortDesc: "Does damage equal to the user's level. Ignores Immunities.",
```
Comment (Zarel) [inline mods/gen1/moves.js]: "immunity" is lowercase. You're not talking about the Ability, after all.

File: mods/gen1/moves.js [hunk 5]
```diff
@@ -664,13 +671,17 @@ exports.BattleMovedex = {
 	},
 	sandattack: {
 		inherit: true,
+		desc: "Lowers the target's accuracy by 1 stage. Ignores Immunities",
+		shortDesc: "Lowers the target's accuracy by 1. Ignores Immunities.",
```
Comment (Zarel) [inline mods/gen1/moves.js]: Lowercase "immunities"

File: mods/gen1/moves.js [hunk 6]
```diff
@@ -664,13 +671,17 @@ exports.BattleMovedex = {
...
 		ignoreImmunity: true,
 		type: "Normal",
 	},
 	seismictoss: {
 		inherit: true,
 		ignoreImmunity: true,
 		basePower: 1,
+		desc: "Deals damage to the target equal to the user's level. Ignores Immunities.",
+		shortDesc: "Does damage equal to the user's level. Ignores Immunities.",
```
Comment (Zarel) [inline mods/gen1/moves.js]: Lowercase "immunities"

File: mods/gen1/moves.js [hunk 7]
```diff
@@ -804,13 +815,17 @@ exports.BattleMovedex = {
 		inherit: true,
 		ignoreImmunity: true,
 		basePower: 1,
+		desc: "Deals damage to the target equal to half of its current HP, rounded down, but not less than 1 HP. Ignores Immunities.",
+		shortDesc: "Does damage equal to 1/2 target's current HP. Ignores Immunities.",
```
Comment (Zarel) [inline mods/gen1/moves.js]: Lowercase "immunities"

File: mods/gen2/items.js [hunk 8]
```diff
@@ -30,6 +30,7 @@ exports.BattleItems = {
 				return 3;
 			}
 		},
+		desc: "If holder is a Chansey, it overrides all crit boosting effects.",
```
Comment (Zarel) [inline mods/gen2/items.js]: That's not even the main thing Lucky Punch does.

It sets Chansey's crit ratio to 3.

File: mods/gen2/items.js [hunk 9]
```diff
@@ -39,6 +40,7 @@ exports.BattleItems = {
 				return 3;
 			}
 		},
+		desc: "If holder is a Farfetch'd, it overrides all crit boosting effects.",
```
Comment (Zarel) [inline mods/gen2/items.js]: You forgot to mention what it sets Chansey's crit ratio to.

File: mods/gen3/items.js (hunk 1 above)
Comment (Marty-D) [inline mods/gen3/items.js]: Why did you remove this? Clearly I meant "when does the holder heal 30 HP?" You can't just say "heals 30 HP", that doesn't mean anything. It's a Berry, it has an activation condition.

Comment (Zarel) [inline mods/gen3/items.js]: Um, you just removed this description instead of fixing it.

File: mods/gen2/items.js (hunk 8 above)
Comment (Marty-D) [inline mods/gen2/items.js]: Also note that PS's stage 3 is actually stage 2 since the most common understanding of critical hit stages is that normal moves start at stage 0.

**Comment by Zarel** [comment]
re: Gen 3 Sitrus Berry

You do realize that Pokémon don't eat berries whenever they feel like it, right? They only eat berries in certain situations. You've neglected that part of the description.

Compare your description: `The holder heals 30 HP.`

With the current description: `Restores 1/4 max HP when at 1/2 max HP or less. Single use.`

Do you see the difference?

File: mods/gen1/moves.js (hunk 3 above)
Comment (DeathlyPlays) [inline mods/gen1/moves.js]: True I had Ignore immunities on it that's why

File: mods/gen2/items.js (hunk 8 above)
Comment (DeathlyPlays) [inline mods/gen2/items.js]: So I say it's 1/3rd?

**Comment by DeathlyPlays** [comment]
@Marty-D @Zarel how about now

**Comment by DeathlyPlays** [comment]
As for Gen Next, should I update it as well?

File: mods/gen3/items.js [hunk 10]
```diff
@@ -216,6 +216,7 @@ exports.BattleItems = {
 		onEat: function (pokemon) {
 			this.heal(30);
 		},
+		desc: "Restores 30 of the holder's health when at 1/2 max HP or less. Single use.",
```
Comment (Zarel) [inline mods/gen3/items.js]: I'd probably say `Restores 30 of the holder's HP when at 1/2 max HP or less. Single use.`

File: mods/gen2/items.js [hunk 11]
```diff
@@ -39,6 +40,7 @@ exports.BattleItems = {
 				return 3;
 			}
 		},
+		desc: "If holder is a Farfetch'd, it overrides all crit boosting effects, and sets Farfetch'd's crit ratio to 3 (crit chance = 1/3).",
```
Comment (Zarel) [inline mods/gen2/items.js]: You need a `shortDesc`, the `desc` is too long for a dex row.

Comment (Zarel) [inline mods/gen2/items.js]: (ditto Chansey)

**Comment by Zarel** [comment]
Feel free to update NEXT descriptions.

File: mods/gen2/items.js (hunk 11 above)
Comment (DeathlyPlays) [inline mods/gen2/items.js]: Alrighty ^~^

**Comment by DeathlyPlays** [comment]
I'll update Gen Next when I get to use a computer unless you accept this between (if so, I can always just make a new PR with the changes for Gen Next)

**Comment by Zarel** [comment]
Well, your latest commit has a build error, so you should probably fix that first...

File: mods/gen2/items.js [hunk 12]
```diff
@@ -30,8 +30,8 @@ exports.BattleItems = {
 				return 3;
 			}
 		},
-		desc: "If holder is a Chansey, it overrides all crit boosting effects, and sets Chansey's crit ratio to 3 (crit chance = 1/3).",
-	},
+		desc: "If holder is a Chansey, it overrides all crit boosting effects, and sets Chansey's crit ratio to 3 (crit chance = 1/3).",	
+		shortDesc: "If holder is a Chansey, it removes all crit boosting effects, and crit ratio = 1/3",
```
Comment (Zarel) [inline mods/gen2/items.js]: Crit ratio is 3, not 1/3

Comment (Zarel) [inline mods/gen2/items.js]: "it removes all crit boosting effects" is also wrong.

The phrasing I'd use would be

`If held by a Chansey, its crit rate is set to 3, overriding all other crit rate changes.`

For a long description, I might clarify that to

`If held by a Chansey, its crit rate is set to 3 (for a crit chance of 33%), overriding all other crit rate changes.`

Comment (DeathlyPlays) [inline mods/gen2/items.js]: Oh

**Comment by DeathlyPlays** [comment]
Done @Zarel

**Comment by DeathlyPlays** [comment]
Fixed eslint

File: mods/gen2/items.js [hunk 13]
```diff
@@ -30,6 +30,8 @@ exports.BattleItems = {
 				return 3;
 			}
 		},
+		desc: "If holder is a Chansey, it overrides all crit boosting effects, and sets Chansey's crit ratio to 3 (crit chance = 1/3).",
+		shortDesc: "If holder is a Chansey, it removes all crit boosting effects, and crit ratio = 3",
```
Comment (Zarel) [inline mods/gen2/items.js]: This is not the desc/shortDesc I told you to use...

Also, the long desc should probably be:

`If held by a Chansey, its critical hit ratio is set to +3 stages (for a crit chance of 33%), overriding all other crit rate changes.`

**Comment by AustinXII** [comment]
"If held by Chansey" also works

**Comment by DeathlyPlays** [comment]
How about now o3o

**Comment by Marty-D** [comment]
Your `shortDesc` is still too long; test them in the teambuilder before deciding if it's usable. Also there are no items with "long descriptions", so everything just uses `desc` and nothing else. I haven't encountered an item yet that's so complicated it requires extra space to describe.

Honestly though, the amount of handholding needed with this PR makes me think I should have closed it as soon as I saw it or I might as well make my own commits on top of yours to fix everything at once since it would take less of everyone's collective time.

**Comment by DeathlyPlays** [comment]
Feel free to do so @Marty-D

**Comment by DeathlyPlays** [comment]
Either close/fix it yourself completely up to you

**Comment by Zarel** [comment]
@Marty-D is there any reason you prefer "If holder is a POKEMON" rather than "If held by POKEMON"?

**Comment by AustinXII** [comment]
I'm pretty sure  "If held by POKEMON" is the correct spelling

**Comment by Marty-D** [comment]
@Zarel Consistency with every other item description that refers to the holder. Finagling "held by" into everything will make them longer. I think it's pretty clear you're supposed to read it as "the holder" anyway; dropping "the" makes all the difference with fitting details in.

**Comment by Zarel** [comment]
I think "If held by" is shorter and clearer, and it might be worth changing every item description for it.

**Comment by DeathlyPlays** [comment]
I was being consistent with the current Description for a majority of these or at least tried

**Comment by Marty-D** [comment]
@Zarel I agree, but only in cases that start with "If the holder is a POKEMON". Obviously I'm not going to change descriptions like `"If holder has a Fire move, this item allows it to use a Fire Z-Move."` or `"If holder's HP is full, will survive an attack that would KO it with 1 HP. Single use."` to use "held by" instead. But yes, I'll consider it with my next major description update coming soon™.
